admin/acceptance.py

```diff
@@ -950,6 +951,101 @@ def eliot_output(message):
     sys.stdout.write(format % message)
     sys.stdout.flush()
 
+def capture_upstart(reactor, host, output_file):
+    """
+    SSH into given machine and capture relevant logs, writing them to
+    output file.
+
+    :param reactor: The reactor.
+    :param bytes host: Machine to SSH into.
+    :param file output_file: File to write to.
+    """
+    # TODO: check the format we need to use here (if we need it at all)
+    # can use it later. Using the whole filename instead of smtg like flocker*
+    # because I can't make it work inside the command. It has a funny way of
+    # parsing the commands. Suggestions are welcome :D
```

Comment:
I don't really know what we need to do regarding formatting, so unfortunately I can't help. 

If you want to use `flocker-*` then you need to run the command via a shell, e.g.

```python
command = [
    b'/bin/sh',
    b'tail',
    b'/var/log/flocker/flocker-*.log',
    b'/var/log/upstart/docker.log',
]
```

This might need some tweaking though.